src/module/loader/module-resolver.cpp

```diff
@@ -153,7 +160,11 @@ ModuleInfo ModuleResolver::ModuleResolverImpl::Resolve(const char* name, const c
     // Normalize module context path.
     filesystem::Path basePath =
         (path == nullptr) ? filesystem::CurrentDirectory() : filesystem::Path(path);
-    
+
+    if (fromContent) {
+        return ResolveFromContent(name, basePath);
```

Comment:
Adding a fromContent flag can be avoided and the function signature will remain clean.

---
In reply to: [133254080](https://github.com/Microsoft/napajs/pull/40#discussion_r133254080) [](ancestors = 133254080)